src/main/java/org/everit/jira/updatenotifier/JTTPVersionUpdater.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (C) 2011 Everit Kft. (http://www.everit.org)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.everit.jira.updatenotifier;
+
+import java.io.IOException;
+
+import org.apache.commons.httpclient.HttpClient;
+import org.apache.commons.httpclient.HttpMethod;
+import org.apache.commons.httpclient.HttpStatus;
+import org.apache.commons.httpclient.methods.GetMethod;
+import org.everit.jira.updatenotifier.exception.UpdateException;
+import org.everit.jira.updatenotifier.json.JiraMarketplaceJSONDTO;
+
+import com.atlassian.jira.component.ComponentAccessor;
+import com.atlassian.jira.util.BuildUtilsInfo;
+import com.google.gson.Gson;
+
+/**
+ * The JTTP version updater class.
+ *
+ */
+public class JTTPVersionUpdater {
```

Comment:
Hmmm. I'm not sure about the JTTP in the name. TimetrackerVersionUpdater would be better.